src/plugins/foxPage/components/OtherOpportunities/FoxOtherOpportunityPanelRow.tsx

```diff
@@ -32,6 +32,27 @@ export const FoxOtherOpportunityPanelRow: React.FC<FoxOtherOpportunityPanelRowPr
     [defiOpportunity, opportunity.link],
   )
 
+  const handleClick = () => {
```

Comment:
Nice one moving this one into its own event handler 💜  but let's memoize it with `useCallback()` - fine-ish not to when touching existing code, but new event handlers should be memoized so we can eventually use memoization lint rules, see https://github.com/shapeshift/web/pull/2698